library/spdm_responder_lib/libspdm_rsp_measurements.c

```diff
@@ -39,76 +71,25 @@ static bool libspdm_create_measurement_signature(libspdm_context_t *spdm_context
         spdm_context->local_context.opaque_measurement_rsp_size +
         signature_size;
     LIBSPDM_ASSERT(response_message_size > measurment_sig_size);
-    ptr = (void *)((uint8_t *)response_message + response_message_size -
-                   measurment_sig_size);
+    fill_response_ptr = (void *)((uint8_t *)response_message + response_message_size -
```

Comment:
Is this `(void *)` cast needed?

Reply:
The `(void *)` is not needed now. I have deleted it.